### ISSUE TITLE
Bug: Searching and updating a keyboard shortcut

### ISSUE THREAD

**Issue by bytebutcher** [bytebutcher/decoder-plus-plus#34]
When opening the dialog to change a keyboard shortcut (here: Help > Keyboard shortcuts) and selecting a keyboard shortcut for actually changing it ... everything works as expected.

However, when filtering keyboard shortcuts by name and changing one the wrong element is selected for change.

Background: Qt works with two models. The source model, and the proxy model. The source model contains all entries while the proxy model does only contain the filtered entries. When clicking an item the item of the proxy model is used which has another index as the one of the source model. 

Long story short: We need to translate proxy model items to source model items...
